pkg/config/example_test.go

```diff
@@ -400,13 +400,22 @@ fUNCdMGmr8FVF6IzTNYGmCuk/C4=
 // baseMSP creates a basic MSP struct for organization.
 func baseMSP() config.MSP {
 	certBlock, _ := pem.Decode([]byte(dummyCert))
-	cert, _ := x509.ParseCertificate(certBlock.Bytes)
+	cert, err := x509.ParseCertificate(certBlock.Bytes)
+	if err != nil {
+		panic(err)
```

Comment:
Why are we panicking instead of passing down the testing.T and using an assertion?